WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -129,6 +129,17 @@ def submit_report_nag(competition)
     )
   end
 
+  def submit_report_reminder(competition)
+    @competition = competition
+    mail(
+      from: Team.weat.email,
+      to: competition.all_delegates.pluck(:email),
+      cc: ["assistants@worldcubeassociation.org"] + delegates_to_senior_delegates_email(competition.all_delegates),
+      reply_to: delegates_to_senior_delegates_email(competition.all_delegates),
```

Comment:
I think you just want the delegates email here: for the "nag" email we want the include the Senior in the reply-all because the delegates are actually late to submit the report, and it may or may not have some consequences.
```suggestion
      reply_to: competition.all_delegates.pluck(:email),
```